hbase-server/src/test/java/org/apache/hadoop/hbase/io/hfile/bucket/TestBucketCache.java

```diff
@@ -255,51 +256,108 @@ public void run() {
 
   @Test
   public void testRetrieveFromFile() throws Exception {
-    HBaseTestingUtility TEST_UTIL = new HBaseTestingUtility();
-    Path testDir = TEST_UTIL.getDataTestDir();
-    TEST_UTIL.getTestFileSystem().mkdirs(testDir);
-
+    Path testDir = HBASE_TESTING_UTILITY.getDataTestDir();
```

Comment:
These two lines for creation of test directory can be consolidated by usage of a private function to reduce code duplication, as there are multiple instances of these?